### DISCUSSION TITLE
licensing

### DISCUSSION THREAD

**Discussion by sonideft** [jfree/jfreechart#322] [General]
If I want to deploy JFreechart in a commercial application and allow users to export the chart in either SVG or PDF format, what level of sponsorship is required. I am just one developer.

**Comment by jfree** [comment]
Hi sonideft, the one-time $100 sponsorship includes a runtime license for JFreeChart, and JFreeSVG or JFreePDF (your choice).  That should meet your requirements but let me know if you have additional questions.

**Reply by jfree** [reply]
Oh, I see you already chose that option - thanks!

**Comment by sonideft** [comment]
Thanks David. I want to use JFreeChart to generate charts in my java app and allow users to right-click and export to SVG or PDF.  I want to make sure I’m compliant with licensing. I build my export menu using isOrsonPDFAvailable(), so it is on my classpath somewhere. Should I sponsor one more time?

Regards,


sonideft

**Comment by jfree** [comment]
> Should I sponsor one more time?

No, I'll make an exception - you can have the runtime license for both JFreeSVG and JFreePDF via the one donation.
